File: SAMER08F.cpp
#include<bits/stc++>
#include<iostream>

void ans(std::vector<int> v)
{
	int t;
	cin>>t;
	if(t!=0)
	{
		int sum=t*(t+1)*(2*t+1)/6;
		fey(v);
		v.push_back(sum);	
	}
}

int main()
{
	std::vector<int> v1;
	std::vector<int> v2;
	std::vector<int> v2;
	ans(&v1,&v2,&v3);
	while(!v.empty())
	{
		cout<<v.pop()<<"\n";
	}
}

File: Random/SAMER08F.py

n=1
t=0
ans=list()
while n is not 0:
	n=int(input())
	if n == 0:
		break
	else:
		t=t+1	
		ans.append(int(n*(n+1)*(2*n+1)/6))

for x in range(0,t-1):
	print(ans.pop(0))
print(ans.pop(-1))